rustfmt.toml

```diff
@@ -57,7 +57,7 @@ skip_children = false
 hide_parse_errors = false
 error_on_line_overflow = false
 error_on_unformatted = false
-report_todo = "Always"
+report_todo = "Never" # changed
```

Comment:
I'd suggest to allow `TODO` in the beginning of the project. And disallow them in a couple of weeks.
But I'm also fine with removing them now and add issues instead.